tests/core/mixins/treeitem.py

```diff
@@ -59,3 +59,16 @@ def test_get_parents(af_tutorial_subdir_po, af_tutorial_po,
 
     parents = afrikaans.directory.get_parents()
     assert len(parents) == 0
+
+
+def test_get_cachekey():
+    from pootle.core.mixins.treeitem import TreeItem
+
+    with pytest.raises(NotImplementedError):
```

Comment:
I wonder how useful it is checking such definitions of the mixin class — it covers none of the core business logic, as opposed to classes which actually make use of `TreeItem`.